### PR TITLE
fix minor grammatical errors and edit for clarity

### PR DESCRIPTION
**Pull request by jaymcgrath**
Hi
I've fixed some grammatical errors and edited a few sections to be more clear and concise. 

*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

## Description

Please describe your pull request.

NOTE: All patches should be made against master, not a maintenance branch like
3.1, 2.5, etc.  That is unless the bug is already fixed in master, but not in
that version series.

If it fixes a bug or resolves a feature request,
be sure to link to that issue via (Fixes #4412) for example.

### CODE REVIEW

File: docs/getting-started/first-steps-with-celery.rst [hunk 1]
```diff
@@ -426,14 +425,14 @@ Worker doesn't start: Permission Error
...
-    make sure that they point to a file/directory that's writable and
+    make sure that they point to a file or directory that's writable and
     readable by the user starting the worker.
 
 Result backend doesn't work or tasks are always in ``PENDING`` state
 --------------------------------------------------------------------
 
 All tasks are :state:`PENDING` by default, so the state would've been
-better named "unknown". Celery doesn't update any state when a task
+better named "unknown." Celery doesn't update the state when a task
```
Comment (georgepsarakis) [inline docs/getting-started/first-steps-with-celery.rst]: I believe the first change in this line should be reverted: `"unknown".` seems the correct version.

File: docs/getting-started/first-steps-with-celery.rst [hunk 2]
```diff
@@ -447,19 +446,19 @@ the task id after all).
...
     an absolute path to make sure this doesn't happen.
 
 4) Make sure the client is configured with the right backend.
 
-    If for some reason the client is configured to use a different backend
-    than the worker, you won't be able to receive the result,
-    so make sure the backend is correct by inspecting it:
+    If, for some reason, the client is configured to use a different backend
+    than the worker, you won't be able to receive the result.
+    Make sure the backend is correct by inspecting it:
```
Comment (georgepsarakis) [inline docs/getting-started/first-steps-with-celery.rst]: Perhaps: 
- `Make sure the backend is configured correctly`
- `Make sure the backend is initializred correctly`

What do you think?

**Comment by georgepsarakis** [comment]
@jaymcgrath can you please rebase?

File: docs/getting-started/first-steps-with-celery.rst (hunk 1 above)
Comment (jaymcgrath) [inline docs/getting-started/first-steps-with-celery.rst]: I can revert it, no problem. 
The difference is stylistic, American convention is punctuation inside quotes, British is outside the quotes. 
http://www.lupinworks.com/roche/pages/quotations.php

**Comment by jaymcgrath** [comment]
Hi George, I rebased and pushed to my fork

**Pull request merged**